hoomd/md/methods/__init__.py

```diff
@@ -7,11 +7,17 @@
 of motion for the system. Each individual method applies the given equations
 of motion to a subset of particles.
 
+.. rubric:: Thermostatted methods
+
+Thermostatted methods require usage of a thermostat, see
+`hoomd.md.methods.thermostats`.
+
 .. rubric:: Integration methods with constraints
 
-For methods that constrain motion to a manifold see `hoomd.md.methods.rattle`
+For methods that constrain motion to a manifold see `hoomd.md.methods.rattle`.
 """
 
 from . import rattle
-from .methods import (Method, Langevin, Brownian, ConstantVolume, ConstantPressure, ConstantEnergy, MTTKThermostat, BussiThermostat,
-                      DisplacementCapped, OverdampedViscous)
+from .methods import (Method, Langevin, Brownian, Thermostatted, ConstantVolume,
+                      ConstantPressure, DisplacementCapped, OverdampedViscous)
+from .thermostats import (Thermostat, Bussi, Berendsen)
```

Comment:
Do we want to expose the thermostats in the method namespace? e.g. `hoomd.md.methods.Bussi`. I think I would want the thermostat namespace.

Reply:
Agreed. Remove this line, users should use `md.methods.thermostats.Bussi`.